mitmproxy/tools/console/overlay.py

```diff
@@ -131,6 +149,11 @@ def keypress(self, size, key):
         elif key in keymap.navkeys:
             return super().keypress(size, key)
 
+    def get_shortcuts(self, choices):
+        if len(self.possible_shortcuts) < len(choices):
+            ctx.log.warn("Too few available shortcuts.")
```

Comment:
It's good that we think of this edge case, but I feel that it's not worth to emit a warning here. A choice without shortcut is still fully functional.